web/template/layout/request-next-case.gotmpl

```diff
@@ -0,0 +1,23 @@
+{{ define "request-next-case" }}
+  {{ if .HasWorkableCase }}
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+        <div class="moj-banner">
```

Comment:
Not a blocker, but we should probably migrate this to the [GOV.UK Notification Banner](https://design-system.service.gov.uk/components/notification-banner/) since [the MOJ one is deprecated](https://moj-design-system.herokuapp.com/components/banner).